recipe/bld.bat

```diff
@@ -1,55 +1,51 @@
-set INCLUDE=%LIBRARY_INC%;%INCLUDE%
-set LIB=%LIBRARY_LIB%;%LIB%
 
-cd qt-everywhere-opensource-src-%PKG_VERSION%
+:: TODO: OpenGL
+set "INCLUDE=%LIBRARY_INC%;%INCLUDE%"
+set "LIB=%LIBRARY_LIB%;%LIB%"
+
+:: this needs to be CALLed due to an exit statement at the end of configure:
+call configure ^
+     -prefix %LIBRARY_PREFIX% ^
+     -libdir %LIBRARY_LIB% ^
+     -bindir %LIBRARY_BIN% ^
+     -headerdir %LIBRARY_INC%\qt ^
+     -archdatadir %LIBRARY_PREFIX% ^
+     -datadir %LIBRARY_PREFIX% ^
+     -L %LIBRARY_LIB% ^
+     -I %LIBRARY_INC% ^
+     -confirm-license ^
+     -no-fontconfig ^
+     -icu ^
+     -no-separate-debug-info ^
+     -no-warnings-are-errors ^
+     -nomake examples ^
+     -nomake tests ^
+     -no-warnings-are-errors ^
+     -opengl desktop ^
+     -opensource ^
+     -openssl ^
+     -platform win32-msvc%VS_YEAR% ^
+     -release ^
+     -shared ^
+     -qt-freetype ^
+     -system-libjpeg ^
+     -system-libpng ^
+     -system-zlib ^
+     -mp
+if errorlevel 1 exit /b 1
+
+:: re-enable echoing which is disabled by configure
+echo on
+     
+:: Note - webengine only built when you ask (nmake module-webengine) - so we can skip it easily.
```

Comment:
Yes I made this note so that it is clear that `webengine` isn't build by default (which we can't use as you say). I've just noticed that `webkit` wasn't being built locally (the sources are included in the Qt tree) so I am just about to follow the instructions [here](https://pempek.net/articles/2015/10/18/compiling-qt-5-5-1-with-visual-studio-2015/) and add `gnuwin32\bin` to the `PATH` to see if that fixes it (`ruby` should already be on AppVeyor). Any other tips appreciated...